.github/ISSUE_TEMPLATE/prerelease_qa.md

```diff
@@ -60,7 +60,7 @@ assignees: ''
 ## JOSM
 - Check authentication
   - [ ] Should be possible to login using OAuth 2
-  - [ ] Should be impossible to login using OAuth 1 or Basic Auth
+  - [ ] ~~Should be impossible to login using OAuth 1 or Basic Auth~~ (OAuth 1 is not working in recent version)
```

Comment:
@Rub21 not sure why you are crossing this one out -- the test is specifically that OAUTH1 **fails** which I think it what you are saying is happening. Maybe we don't need this test at all anymore, so long as OAUTH 2 works?